build.sbt

```diff
@@ -7,19 +7,47 @@ ThisBuild / scmInfo := Some(
   ScmInfo(url("https://github.com/avast/scala-server-toolkit"), "scm:git:git@github.com:avast/scala-server-toolkit.git")
 )
 
-ThisBuild / scalaVersion := "2.13.1"
+ThisBuild / scalaVersion := "2.13.0"
```

Comment:
why not 2.13.1?

Reply:
Because `2.13.1` does not work with `mdoc`.